src/LIBSVM.jl

```diff
@@ -345,6 +351,7 @@ function svmtrain(
         weights = Float64[]
         reverse_labels = Bool[]
     else
+        check_dims(X, y, kernel)
```

Comment:
```suggestion
        @boundscheck check_dims(X, y, kernel)
```

Reply:
Is this an appropriate use of `@boundscheck`? We do not check any bounds, we just validate the input.

Reply:
Ah, you're right. Just ignore that suggestion.